contracts/libs/Mappings.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MIT
+// heavily inspired by: https://bitbucket.org/rhitchens2/soliditystoragepatterns/src/master/GeneralizedCollection.sol
+pragma solidity ^0.8.8;
+
+import "./Debug.sol"; // DELETE ME
+
+library Mappings {
+  type KeyId is bytes32;
+  type ValueId is bytes32;
+
+  // first entity is called a "One"
```

Comment:
Might as well remove these comments; they are no longer up to date, as these type of comments tend to do ;)